python/ray/serve/backend_state.py

```diff
@@ -111,39 +123,69 @@ def start_or_update(self, backend_info: BackendInfo):
                     lifetime="detached" if self._detached else None,
                     name=self._placement_group_name)
 
-        try:
-            self._actor_handle = ray.get_actor(self._actor_name)
-        except ValueError:
-            self._actor_handle = None
-
-        if self._actor_handle is None:
-            logger.debug("Starting replica '{}' for deployment '{}'.".format(
-                self._replica_tag, self._backend_tag) +
-                         f" component=serve deployment={self._backend_tag} "
-                         f"replica={self._replica_tag}")
-
-            self._actor_handle = backend_info.actor_def.options(
-                name=self._actor_name,
-                lifetime="detached" if self._detached else None,
-                placement_group=self._placement_group,
-                placement_group_capture_child_tasks=False,
-                **backend_info.replica_config.ray_actor_options).remote(
-                    self._backend_tag, self._replica_tag,
-                    backend_info.replica_config.init_args,
-                    backend_info.backend_config, self._controller_name,
-                    self._detached)
+    def start(self, backend_info: BackendInfo, version: BackendVersion):
+        """
+        Start a new actor for current BackendReplica instance.
+        """
+        self._handle_placement_group(backend_info)
+
+        logger.debug(f"Starting replica {self.replica_tag} for deployment "
+                     f"{self.backend_tag} component=serve deployment="
+                     f"{self.backend_tag} replica={self.replica_tag}")
+
+        self._actor_handle = backend_info.actor_def.options(
+            name=self._actor_name,
+            namespace=self._controller_namespace,
+            lifetime="detached" if self._detached else None,
+            placement_group=self._placement_group,
+            placement_group_capture_child_tasks=False,
+            **backend_info.replica_config.ray_actor_options).remote(
+                self.backend_tag, self.replica_tag,
+                backend_info.replica_config.init_args,
+                backend_info.backend_config, version, self._controller_name,
+                self._detached)
 
         self._ready_obj_ref = self._actor_handle.reconfigure.remote(
             backend_info.backend_config.user_config)
 
-    def check_ready(self) -> ReplicaStartupStatus:
+    def update(self, user_config: Any):
+        """
+        Update user config of existing actor behind current
+        BackendReplica instance.
+        """
+        logger.debug(f"Updating replica {self.replica_tag} for deployment "
+                     f"{self.backend_tag} component=serve deployment="
+                     f"{self.backend_tag} replica={self.replica_tag} "
+                     f"with user_config {user_config}")
+
+        self._actor_handle = ray.get_actor(
+            self._actor_name, namespace=self._controller_namespace)
```

Comment:
can we cache the actor handle so we don't always have to do this? otherwise we make an RPC to GCS every timie.